control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -2022,6 +2022,12 @@ func (r *HostedControlPlaneReconciler) reconcilePKI(ctx context.Context, hcp *hy
 			return fmt.Errorf("failed to reconcile ignition server cert: %w", err)
 		}
 	}
+	paoWebhookSecret := manifests.PerformanceAddonOperatorWebhookCertSecret(hcp.Namespace)
```

Comment:
Should we keep using the Performance Addon Operator name or should we move to Performance Profile Controller? (or anything better suited - here and below)

Reply:
I agree with the change in general, but I would prefer to wait and do it once everything else is working, maybe even in a different PR .

Reply:
I'm skeptical a trivial rename can break things badly, but I'm also fine deferring the cleanup to a followup PR

Reply:
changed